.github/workflows/check-available-os-versions.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/sh
+
+set -e
+
+SURPRISE_COUNT=0
+[ -z "${HTML2TEXT}" ] && HTML2TEXT=html2text
+
+fetch_webpage_as_markdown() {
+	curl -sS "$1" | ${HTML2TEXT}
+}
+
+fetch_alpine_upstream_latest_stable() {
+	fetch_webpage_as_markdown 'https://alpinelinux.org/downloads/' \
+		| grep 'Current Alpine Version' \
+		| awk '{print $4}' \
+		| sed -e 's|*||g'
```

Comment:
Is this really stripping `*`? I thought this would need to be escaped like `\*`.
No trouble with that though.

Reply:
Yes, I was surprised to see it work as is. Which is precisely why escaping is probably better. I'll test and commit if so.